### PR TITLE
Change results processing to handle new FPG and BMI stratification.

### PR DESCRIPTION
**Pull request by KjellSwedin**
Add  changes to deal with new FPG and BMI stratification. Remove ACS as it was not an original requirement. Correct a couple small issues that were not apparent until running the results processor. Standardize on normal|high versus (normal|low)|high.
Scroll right to see the new stratification.

|  | age | sex | year | cause | measure | input_draw | scenario | value | SBP | LDL | FPG | BMI
|  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
| 0 | 25_to_29 | female | 2021 | acute_ischemic_stroke | state_person_time | 0 | baseline | 0 | high | high | high | high
| 1 | 25_to_29 | female | 2021 | acute_ischemic_stroke | state_person_time | 0 | baseline | 0 | high | high | high | normal
| 2 | 25_to_29 | female | 2021 | acute_ischemic_stroke | state_person_time | 0 | baseline | 0 | high | high | normal | high

### CODE REVIEW

File: src/vivarium_nih_us_cvd/components/observers.py [hunk 1]
```diff
@@ -89,6 +83,7 @@ def on_initialize_simulants(self, pop_data: 'SimulantData'):
             id_str = '_'.join([j.id for j in group])
             risk_groups.loc[mask] = id_str
 
+        self.risk_group_names = risk_groups.unique()
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/components/observers.py]: Wont this miss any stratifications that happen not to have any simulants in it? Do you want the list of stratifications which can exist, or the stratifications which actively exist on simulation?

**Review by rmudambi — APPROVED** [review]

File: src/vivarium_nih_us_cvd/components/observers.py (hunk 1 above)
Comment (KjellSwedin) [reply src/vivarium_nih_us_cvd/components/observers.py]: Good catch, I think you are correct. It doesn't matter with my current configuration (have simulants in all strata) but could fail under different circumstances.

Comment (KjellSwedin) [reply src/vivarium_nih_us_cvd/components/observers.py]: Addressed.

File: src/vivarium_nih_us_cvd/components/observers.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ class ResultsStratifier:
 
     def __init__(self, observer_name: str):
         self.name = f'{observer_name}_results_stratifier'
+        self._risk_group_ids = []
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/components/observers.py:39]: Is there a reason you're defining this in the `init`?

**Review by rmudambi — APPROVED** [review]

**Pull request merged**